src/modules/Search/Search.d.ts

```diff
@@ -30,7 +30,7 @@ export interface SearchProps {
   noResultsDescription?: string;
```

Comment:
@b0gok Thanks for a PR. I think that we need make same changes for `noResultsDescription`.